src/styles/components/_component-card.scss

```diff
@@ -1,79 +1,84 @@
-.component-link {
-  display: block;
-  outline: none;
+.component-card {
+  display: flex;
+  flex-direction: column-reverse;
+  height: 21.875rem; // 350px/16
   width: 100%;
-  height: 100%;
+  cursor: pointer;
+  transition: all 150ms ease-in-out;
+  border: 2px solid var(--foreground-color);
 
-  .component-card {
-    display: flex;
-    flex-direction: column-reverse;
-    height: 21.875rem; // 350px/16
-    width: 100%;
-    cursor: pointer;
+  .component-link:focus & {
+    outline: none;
+    border: 2px solid var(--accent-color);
+  }
+
+  &__image {
+    box-sizing: border-box;
     transition: all 150ms ease-in-out;
-    border: 2px solid var(--foreground-color);
+    width: 100%;
+    height: 70%;
+    padding: min(15%, 2rem);
+    margin: auto;
+  }
 
-    &__image {
-      box-sizing: border-box;
-      transition: all 150ms ease-in-out;
-      width: 100%;
-      height: 70%;
-      padding: min(15%, 2rem);
-      margin: auto;
-    }
+  @media (prefers-color-scheme: dark) {
+    border: 2px solid var(--background-color);
+  }
 
-    @media (prefers-color-scheme: dark) {
-      border: 2px solid var(--background-color);
-    }
+  @media (min-width: $breakpoint-scale-up) {
+    height: 100%;
+  }
 
-    @media (min-width: $breakpoint-scale-up) {
-      height: 100%;
-    }
+  &__image-open {
+    display: block;
 
-    &:hover img {
-      scale: 102%;
+    .component-link:hover & {
+      display: none;
     }
+  }
 
-    &__header {
-      font-family: Oxygen, sans-serif;
-      font-weight: 700;
-      z-index: 1;
-      font-size: map.get($font-sizes, "sm");
-      line-height: 4.6667rem; // 75px
-      letter-spacing: 0.02em;
-      background-color: var(--light-background-color);
-      position: relative;
-      text-align: center;
+  &__image-closed {
+    display: none;
+
+    .component-link:hover & {
       display: block;
-      border-top: 0.25rem solid var(--foreground-color);
+    }
```

Comment:
Same thing here: I pasted those styles originally from `&__image` and put them on this `&__image-closed` hover.